packages/plugins/Web3Profile/src/SNSAdaptor/assets/arrow.tsx

```diff
@@ -0,0 +1,13 @@
+import { createIcon } from '@masknet/icons'
```

Comment:
Move this file to `package/icons`.

Reply:
```package/icons``` has Arrow icon, which is different from what we used in this plugin

Reply:
Does that icon in use now?